pkg/trace/traceutil/trace.go

```diff
@@ -49,6 +50,22 @@ func GetAppVersion(root *pb.Span, t *pb.TraceChunk) string {
 	return ""
 }
 
+// GetGitCommitSha returns the first "git_commit_sha" tag found in trace t.
+func GetGitCommitSha(root *pb.Span, t *pb.TraceChunk) string {
```

Comment:
I'm looking into another source for git commit sha - as a container tag: https://docs.datadoghq.com/containers/docker/tag/?tab=containerizedagent#out-of-the-box-tagging